backend/tests/jest.setup.cjs

```diff
@@ -1,13 +1,13 @@
 require('dotenv').config();
-const MongoMemoryServer = require('mongodb-memory-server').MongoMemoryServer;
+const { MongoMemoryReplSet } = require('mongodb-memory-server');
 const mongoose = require('mongoose');
 
 module.exports = async () => {
   console.log('\nSetup: config mongodb for testing...');
   // https://typegoose.github.io/mongodb-memory-server/docs/guides/integration-examples/test-runners/
-  const instance = await MongoMemoryServer.create();
-  const uri = instance.getUri();
-  global.__MONGOINSTANCE = instance;
+  const replSet = await MongoMemoryReplSet.create({ replSet: { count: 3 } });
+  const uri = replSet.getUri();
+  global.__MONGOREPLSET = replSet;
   process.env.MONGO_URI = uri.slice(0, uri.lastIndexOf('/'));
```

Comment:
I think it's a good instinct to want to keep environment variables clean, but I'm not really seeing the benefit of doing this change.  The environment variable only changes in the jest process, and that's terminating after the teardown script completes, so nothing else is going to use the original MONGO_URI, and it won't persist.

One improvement I could see as beneficial would be to figure out a way to choose between multiple test DBs. Maybe some test environments will use the memory db, but some test environments might use a dedicated MongoDB replica set. Currently, all test environments are going to use an in-memory DB. I created issue #202 for this.

I'd rather not make this change here because it seems out of scope from the main part of this PR.

Reply:
Thank you for creating that Issue! Let's leave it as you had it and defer this plus the changes you suggested in #202 to another PR later on. I left a comment in there suggesting a potential solution.